tools/mptcpify.txt

```diff
@@ -0,0 +1,21 @@
+Demonstrations of mptcpify, the Linux eBPF/bcc version.
```

Comment:
Please change mptcpify.txt to mptcpify_example.txt.

Please also add mptcpify_example.txt to README.md file at the root directory.